php/README.md

```diff
@@ -0,0 +1,7 @@
+# MailChimp API v3.0 Examples for PHP
+
+These examples of MailChimp API v3.0 implementation in PHP have been tested with PHP 5.5.12.
+
+Require [cURL](http://php.net/manual/en/book.curl.php) and PHP.
+
+To add a subscriber to a list, replace the values of the method call in `add-subscriber.php` by your own ones.
```

Comment:
Also mention to replace `apiUser` with your username. I forgot about that while trying the library!